docs/identity-platform/apple-sso-plugin.md

```diff
@@ -433,7 +433,10 @@ The list below contains some common scenarios that will be impacted by these cha
 This is not an exhaustive list and we do advise both consumers and vendors of applications to test their software for compatibility with this new datastore.
 
 #### Registered/Enrolled Device Conditional Access Policy Support in Chrome
-To support device Conditional Access policies in Google Chrome with Secure Enclave based storage enabled, you will need to have the [Windows Accounts](https://chromewebstore.google.com/detail/windows-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji) extension installed and enabled.
+To support device Conditional Access policies in Google Chrome with Secure Enclave based storage enabled, you will need to have the [Microsoft Single Sign On](https://chromewebstore.google.com/detail/windows-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji) extension installed and enabled.
+
+1. This is required for Mac devices that enable Platform SSO and for devices that have opted-in to Secure Enclave backed device identity (https://learn.microsoft.com/en-us/entra/identity-platform/apple-sso-plugin#registeredenrolled-device-conditional-access-policy-support-in-chrome).
```

Comment:
You can omit bullet \#1 here since the surrounding section of this document covers this.

I'd suggest turning \#2 into a Note like...

> **Tip:** Admins can automatically deploy Chrome Extensions using MDM ([learn more](https://chromeenterprise.google/policies/?policy=ExtensionInstallForcelist)).

Reply:
Just committed, please review when you can